index.js

```diff
@@ -118,8 +118,12 @@ DJI_SRT_Parser.prototype.millisecondsPerSample = function (metadata, millisecond
 };
 
 DJI_SRT_Parser.prototype.interpretMetadata = function (arr, smooth) {
-  // Forcing srt to have one information line plus the timecode. Preventing empty lines and incomplete data in the array, something frequent at the end of the DJI´s SRTs.
-  arr = arr.filter((value) => Object.keys(value).length > 1);
+
+  // Forcing srt to have two information lines plus the timecode. Preventing empty lines and incomplete data in the array, something frequent at the end of the DJI´s SRTs.
```

Comment:
Mavic mini (and other models that embed the data in the mp4) does not include timecode in the SRT content. I will try to find some samples to add tests

Reply:
Nevermind, I think it's fine, as the timecode data will be deduced when extracting the subtitles. I will provide a Mavic Mini sample anyway.